.github/workflows/python-package.yaml

```diff
@@ -54,6 +54,11 @@ jobs:
         python -m pip install --upgrade pip
         python -m pip install -e .
         python -c "import triton_cli; print(triton_cli.__version__)"
+    - name: Install dependencies (GenAI-Perf)
+      run: |
+        git clone --depth 1 https://github.com/triton-inference-server/client.git
+        cd client/src/c++/perf_analyzer/genai-perf
+        pip install .
```

Comment:
Instead of installing this here, can you see if it's possible to pip install directly from git as a dependency in the pyproject.toml?

`pip install git+http://.../genai-perf` -- I wonder if the "git+http..." part can be used directly as a "dependency" just like how tritonclient and other package dependencies are currently getting installed in CLI

Reply:
Interesting idea! I can try that approach. Thanks for suggesting it.

Reply:
Done! This PR has gotten quite small, so I'm going to merge it into the original PR then prepare that one for review once I remove the PA/MA pieces there as well.